src/pages/Vote/index.tsx

```diff
@@ -152,14 +153,13 @@ export default function Vote() {
               </RowBetween>
               <ExternalLink
                 style={{ color: 'white', textDecoration: 'underline' }}
-                href="https://uniswap.org/blog/uni"
+                href="http://wwww.google.com"
```

Comment:
can just link to the main braintrust website for now until we have our blog/docs up